Classes/HRLProj.uc

```diff
@@ -3,6 +3,7 @@ class HRLProj extends ScrnLAWProj;
 defaultproperties
 {
     ExplosionClass=Class'ScrnWeaponPack.HRLExplosion'
+    ImpactDamage=275
```

Comment:
Good change. It makes sense that rockets do more impact damage than grenades, and LAW has a stronger impact than HRL.

Reply:
this change is also quite significant, previously demo perked headshots would only do 639, just barely enough to stun a scrake. After this change HRL will be able to stun scrakes and LAW will be able to stun jasons, this makes a big difference for demos engaging charging scrakes within arming range by allowing a stun with a blunt headshot which was only possible with RPG before due to 500 base impact damage (which is now lowered to 400)
The blunt damage change was to make the other launchers consistent with the RPG, but if this makes demos vs charging scrakes too easy perhaps all impact damage will have to be tweaked